adapters/rubicon/rubicon.go

```diff
@@ -852,12 +852,14 @@ func (a *RubiconAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *ada
 		if request.Site != nil {
 			siteCopy := *request.Site
 			siteExtRP := rubiconSiteExt{RP: rubiconSiteExtRP{SiteID: rubiconExt.SiteId}}
-			target, err := updateExtWithIabAttribute(nil, siteCopy.Content.Data, []int{1, 2})
-			if err != nil {
-				errs = append(errs, err)
-				continue
+			if siteCopy.Content != nil {
+				target, err := updateExtWithIabAttribute(nil, siteCopy.Content.Data, []int{1, 2})
```

Comment:
We need to verify `siteCopy.Content` is not `nil` before referencing `siteCopy.Content.Data`.